app/assets/stylesheets/modules/footer.scss

```diff
@@ -3,14 +3,17 @@
 .footer {
   background: $blue;
   bottom: 0;
+  left: 0;
```

Comment:
Properties should be ordered background, bottom, color, height, left, line-height, position, text-align, width